fe/fe-core/src/main/java/com/starrocks/load/InsertOverwriteJob.java

```diff
@@ -0,0 +1,104 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.load;
+
+import com.google.gson.annotations.SerializedName;
+import com.starrocks.analysis.InsertStmt;
+
+import java.util.List;
+
+public class InsertOverwriteJob {
+    @SerializedName(value = "jobId")
+    private long jobId;
+
+    @SerializedName(value = "jobState")
+    private InsertOverwriteJobState jobState;
+
+    @SerializedName(value = "sourcePartitionNames")
+    private List<String> sourcePartitionNames;
+
+    @SerializedName(value = "newPartitionNames")
+    private List<String> newPartitionNames;
+
+    @SerializedName(value = "targetDbId")
+    private long targetDbId;
+
+    @SerializedName(value = "targetTableId")
+    private long targetTableId;
+
+    @SerializedName(value = "originalTargetPartitionIds")
+    private List<Long> originalTargetPartitionIds;
+
+    private InsertStmt insertStmt;
```

Comment:
if the insertStmt is not serialized. how can the job rerun after FE restart.

Reply:
Insert overwrite will be cancelled after FE restart, so do not need to serialize the insertStmt